apps/desktop/src/renderer/pages/settings/sections/Section.module.css

```
@@ -646,11 +646,11 @@
 }
 
 /* Spinner animation for loading states */
-.plugin-spinner {
-  animation: plugin-spin 1s linear infinite;
+.pluginSpinner {
+  animation: pluginSpin 1s linear infinite;
 }
 
-@keyframes plugin-spin {
+@keyframes pluginSpin {
```

Stylelint (17.9.0)

[error] Expected keyframe name "pluginSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)